packages/node_modules/samples/browser-call-with-screenshare/app.js

```diff
@@ -263,13 +262,172 @@ function joinMeeting(meeting) {
   });
 }
 
-// In order to simplify the state management needed to keep track of our button
-// handlers, we'll rely on the current meeting global object and only hook up event
-// handlers once.
+function handleRadio() {
+  // checks which radio is enabled
+  isPaste = document.getElementById('paste').checked;
+  isOAuth = document.getElementById('oauth').checked;
+
+  const div = document.createElement('div');
+
+  // removes the child node of the form if anything exists
+  while (document.getElementById('authForm').firstChild) {
+    document.getElementById('authForm').removeChild(document.getElementById('authForm').firstChild);
+  }
+
+  // adds the correct form based on enabled radio
+  if (isOAuth) {
+    const legend = document.createElement('legend');
+    const login = document.createElement('input');
+    const logout = document.createElement('input');
+    const loginStatus = document.createElement('p');
+
+
+    legend.innerText = 'Authentication with OAuth';
+    loginStatus.innerText = 'not logged in';
+
+    login.setAttribute('id', 'login');
+    login.setAttribute('title', 'login');
+    login.setAttribute('type', 'submit');
+    login.setAttribute('value', 'login');
+    logout.setAttribute('id', 'logout');
+    logout.setAttribute('title', 'logout');
+    logout.setAttribute('type', 'submit');
+    logout.setAttribute('value', 'logout');
+    logout.setAttribute('disabled', 'disabled');
+    loginStatus.setAttribute('id', 'login-status');
+    div.append(legend);
+    div.append(login);
+    div.append(logout);
+    div.append(loginStatus);
+
+    // div.innerText = `
+    //     <legend>Authentication with OAuth </legend>
+    //           <input id="login" title="login" type="submit" value="login">
+    //           <input id="logout" title="logout" type="submit" value="logout" disabled="disabled">
+    //           <p id="login-status">not logged in</p>
+    // `;
+
+    document.getElementById('authForm').appendChild(div);
+
+    // Now, let's set up oauth login
+    document.getElementById('credentials').addEventListener('submit', (event) => {
+    // let's make sure we don't reload the page when we submit the form
+      event.preventDefault();
+
+      // Initiate login to Webex account
+      webex.authorization.initiateLogin();
+    });
+
+    // Now, let's set up logout
+    document.getElementById('logout').addEventListener('click', (event) => {
+      // let's make sure we don't reload the page when we submit the form
+      event.preventDefault();
+
+      if (webex.canAuthorize) {
+        // if already authenticated, logout on click
+        webex.logout();
+      }
+    });
+  }
+
+  if (isPaste) {
+    const legend = document.createElement('legend');
+    const accessToken = document.createElement('input');
+    const webexInit = document.createElement('input');
+    const register = document.createElement('input');
```

Comment:
Don't forget, the register button needs to be used by both authentication types.